lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -368,6 +373,27 @@ function (ClassMetadata $class) use ($idMapping) : bool {
         return $schema;
     }
 
+    /**
+     * @param Schema $schema
+     * @param Table  $table
+     *
```

Comment:
No need for such phpdoc when you have type declarations.

```suggestion
```

Reply:
I did it to keep the file homogeneous